src/main.cpp

```diff
@@ -143,18 +93,16 @@ auto main( int argc, char* argv[] ) -> int {
 		excludes.emplace_back( ".*\\.vmx" );
 		excludes.emplace_back( ".*\\.log" );
 		excludes.emplace_back( ".*verifier_index\\.rsv" );
-		ret = create( root, indexLocation, excludes, overwrite, output.get() );
+		ret = create( root, indexLocation, excludes, overwrite );
 	} else {
 		// warn about stuff which shouldn't be here, don't use the `output::report` as this is a negligible user error
```

Comment:
i'd update the comment too

Reply:
fixed it!

Reply:
its still the same?

Reply:
looks correct on my screen? refresh the page?

Reply:
oh yeah, the ui didn't update, my bad